Updation.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+require 'Database.php';
+require 'creds.php';
+
+session_start();
```

Comment:
Let's not do this here.
Ideally the class file itself should contain the class and it's functions only.